missing `zero()` function on ARM with `linux_5_11` feature

Trying to enable `linux_5_11` feature and building for an ARM32 platform fails with

```
   Compiling rustix v1.1.2
error[E0425]: cannot find function `zero` in this scope
   --> .../rustix-1.1.2/src/backend/linux_raw/fs/syscalls.rs:296:13
    |
296 |             zero(),
    |             ^^^^ not found in this scope
    |
help: consider importing this function
    |
  9 + use crate::backend::conv::zero;
```


`zero()` is imported only certain features are not enabled

https://github.com/bytecodealliance/rustix/blob/4e316aec7f6f4a1ecb53cd76e74dd7431e1e8360/src/backend/linux_raw/fs/syscalls.rs#L11-L18

but used with a different feature set later

https://github.com/bytecodealliance/rustix/blob/4e316aec7f6f4a1ecb53cd76e74dd7431e1e8360/src/backend/linux_raw/fs/syscalls.rs#L283-L296

--- a/src/backend/linux_raw/fs/syscalls.rs
+++ b/src/backend/linux_raw/fs/syscalls.rs
@@ -14,6 +14,13 @@ use crate::backend::conv::fs::oflags_for_open_how;
     target_arch = "riscv64",
     target_arch = "mips",
     target_arch = "mips32r6",
+    all(
+        target_pointer_width = "32",
+        any(
+            target_arch = "arm",
+            target_arch = "powerpc"
+        ),
+    )
 ))]
 use crate::backend::conv::zero;
 use crate::backend::conv::{

